src/components/Editor/Editor.svelte

```diff
@@ -17,19 +17,21 @@
 </script>
 
 <style lang="scss">
-	:global(pre::selection, code::selection, code ::selection) {
+	:global(pre::selection, code::selection, code::-moz-selection, code *::selection) {
```

Comment:
@Valexr could we drop `-moz-`? I though, it is not used anymore.

Reply:
removed 👌🏻